### DISCUSSION TITLE
DetailsList missing in v9?

### DISCUSSION THREAD

**Discussion by foxxycodes** [microsoft/fluentui#24017] [Q&A] [answered]
Is the `DetailsList` component not implemented yet in v9 or will it be deprecated as a whole?

**Comment by j3rrym0use** [comment, marked as answer]
https://github.com/microsoft/fluentui/wiki/Fluent-UI-React-v9-Component-Roadmap

check this out, `DetailList` could be `Table` or some how.

it's not avaliable for now :)
